packages/client/wallets/aa/src/passkeys/services/LitService.ts

```diff
@@ -0,0 +1,204 @@
+import { logError, logInfo } from "@/services/logging";
+import { LitProtocolError, errorToJSON, isLocalhost } from "@/utils";
+import { POLYGON_CHAIN_ID, RELAY_API_KEY } from "@/utils/constants";
+import { LitAbility, LitActionResource } from "@lit-protocol/auth-helpers";
+import { ProviderType } from "@lit-protocol/constants";
+import { LitAuthClient, WebAuthnProvider } from "@lit-protocol/lit-auth-client";
+import { LitNodeClient, decryptToString, encryptString } from "@lit-protocol/lit-node-client";
+import { PKPEthersWallet } from "@lit-protocol/pkp-ethers";
+import { AuthSig } from "@lit-protocol/types";
+import { SiweMessage } from "siwe";
+
+const chain = "polygon";
+
+export class LitService {
+    private litNodeClient: LitNodeClient | undefined;
+    private litAuthClient: LitAuthClient | undefined;
+
+    async connect() {
+        this.litAuthClient = new LitAuthClient({
+            litRelayConfig: {
+                relayApiKey: RELAY_API_KEY,
+            },
+        });
+        this.litAuthClient.initProvider<WebAuthnProvider>(ProviderType.WebAuthn);
+        this.litNodeClient = new LitNodeClient({
+            litNetwork: isLocalhost() ? "manzano" : "habanero",
+        });
+        await this.litNodeClient.connect();
+    }
+
+    async registerWithWebAuthn(identifier: string) {
+        try {
+            if (this.litAuthClient == null) {
+                await this.connect();
+            }
+            const provider = this.litAuthClient!.getProvider(ProviderType.WebAuthn) as WebAuthnProvider;
```

Comment:
we just checked `litAuthClient == null` so we should be able to get rid of the `!`
```suggestion
            const provider = this.litAuthClient.getProvider(ProviderType.WebAuthn) as WebAuthnProvider;
```